### PR TITLE
Feat: Onboard Uniref50 dataset

### PR DESCRIPTION
**Pull request by gkodukula**
## Description

Dataset: Uniref50

## Checklist

- [x] **(Required)** This pull request is appropriately labeled
- [x] Please merge this pull request after it's approved

Use the sections below based on what's applicable to your PR and delete the rest:

### Feature
- [x] I'm adding or editing a feature
- [x] I have updated the [`README`](https://github.com/GoogleCloudPlatform/public-datasets-pipelines/blob/main/README.md) accordingly
- [x] I have added/revised tests for the feature

### Data Onboarding
- [x] I'm adding or editing a dataset
- [x] The [Google Cloud Datasets team](mailto:cloud-datasets-onboarding@google.com) is aware of the proposed dataset
- [x] I put all my code inside  `datasets/<Uniref50>` and nothing outside of that directory

### CODE REVIEW

**Comment by gkodukula** [comment]
@nlarge-google can you please review this code

File: datasets/uniref50/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+    target_file: pathlib.Path,
+    project_id: str,
+    dataset_id: str,
+    table_id: str,
+    schema_path: str,
+    chunksize: str,
+    target_gcs_bucket: str,
+    target_gcs_path: str,
+) -> None:
+
```
Comment (nlarge-google) [inline datasets/uniref50/pipelines/_images/run_csv_transform_kub/csv_transform.py:41]: Please remove extra lines

File: datasets/uniref50/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+    target_file: pathlib.Path,
+    project_id: str,
+    dataset_id: str,
+    destination_table: str,
+    schema_path: str,
+    chunksize: str,
+    target_gcs_bucket: str,
+    target_gcs_path: str,
+) -> None:
+
```
Comment (nlarge-google) [inline datasets/uniref50/pipelines/_images/run_csv_transform_kub/csv_transform.py:75]: Please remove extra lines

**Review by nlarge-google — CHANGES_REQUESTED** [review]

File: datasets/uniref50/pipelines/_images/run_csv_transform_kub/csv_transform.py (hunk 2 above)
Comment (gkodukula) [reply datasets/uniref50/pipelines/_images/run_csv_transform_kub/csv_transform.py]: done @nlarge-google

File: datasets/uniref50/pipelines/_images/run_csv_transform_kub/csv_transform.py (hunk 1 above)
Comment (gkodukula) [reply datasets/uniref50/pipelines/_images/run_csv_transform_kub/csv_transform.py]: done @nlarge-google

**Review by nlarge-google — APPROVED** [review]

**Pull request merged**